File: android/src/main/kotlin/com/jameskarl/amap/map/apis/FlutterApi.kt
package com.jameskarl.amap.map.apis

import com.amap.api.maps.AMap
import com.jameskarl.amap.map.ReplyToFlutter

interface FlutterApi {
    fun handle(methodId: String, map: AMap, data: Any?): ReplyToFlutter?
}

File: ios/Classes/apis/FlutterApi.swift
//
//  FlutterApi.swift
//  amap
//
//  Created by James Karl on 2019/9/12.
//

import Foundation
import MAMapKit

protocol FlutterApi {
    func handle(mapView: MAMapView, methodId: String, data: Any?) -> ReplyToFlutter?
}
